yt_dlp/extractor/facebook.py

```diff
@@ -659,7 +674,8 @@ def parse_attachment(attachment, key='media'):
                 for edge in edges:
                     parse_attachment(edge, key='node')
 
-                video = data.get('video') or {}
+                video = (traverse_obj(data, ('event', 'cover_media_renderer', 'cover_video', {dict}))  # event cover video
+                         or data.get('video') or {})
```

Comment:
it's not good to do `traverse_obj(data, ...) or data.get(...)` when we can pass multiple paths to `traverse_obj` instead
```suggestion
                video = traverse_obj(data, (
                    'event', 'cover_media_renderer', 'cover_video'), 'video', expected_type=dict) or {}
```